ops/Dockerfile

```diff
@@ -44,7 +44,14 @@ RUN gpuci_mamba_retry env update -f /environment.yml \
     && rm /environment.yml
 
 FROM base-test-install as wheel-install-0
-RUN conda run --no-capture-output -n triton_test pip install tritonclient[all]
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+      build-essential \
+      ca-certificates \
+      git \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && conda run --no-capture-output -n triton_test pip install tritonclient[all]
```

Comment:
`tritonclient[all]` has dependency constraint `geventhttpclient<=2.0.2`, but there isn't a binary wheel for geventhttpclient 2.0.2 that targets Python 3.12 (version used in the base Triton container). So we need `build-essential` to be able to install `geventhttpclient` from the source distribution.

Reply:
Oof, that's unfortunate, but no way around it. Seems fine by me.